apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1/FilterPartial.cs

```diff
@@ -62,6 +62,34 @@ public static Filter And(IEnumerable<Filter> filters)
         public static Filter Equal(string propertyName, Value propertyValue) =>
             Property(propertyName, propertyValue, Operator.Equal);
 
+        /// <summary>
+        /// Creates a filter to check that the specified property is not equal to a given value.
+        /// </summary>
+        /// <param name="propertyName">The name of the property. Must not be null.</param>
+        /// <param name="propertyValue">The value to compare against. May be null, which indicates
+        /// a value with <see cref="Value.NullValue"/> set.</param>
+        /// <returns>The newly created filter.</returns>
+        public static Filter NotEqual(string propertyName, Value propertyValue) =>
+            Property(propertyName, propertyValue, Operator.NotEqual);
+
+        /// <summary>
+        /// Creates a filter to check that the specified property is in a given array of values.
+        /// </summary>
+        /// <param name="value">The name of the property. Must not be null.</param>
+        /// <param name="collection">The array of values to compare against. </param>
```

Comment:
Can this be null? If so, what's the meaning? If not, we should probably validate it. (Ditto for NotIn.)

Reply:
(Apologies for not catching this first time round...)